tasks/configure.yml

```diff
@@ -56,6 +56,21 @@
   notify:
     - reload pgbouncer
 
+- name: Divert pgbouncer default file /etc/default/pgbouncer
+  command: dpkg-divert --quiet --local --divert /etc/default/pgbouncer.dpkg-divert --rename /etc/default/pgbouncer
+  args:
+    creates: "/etc/default/pgbouncer.dpkg-divert"
+
+- name: pgbouncer default file /etc/default/pgbouncer
+  template:
+    src: pgbouncer_default.j2
+    dest: "/etc/default/pgbouncer"
+    owner: "{{ pgbouncer_user }}"
```

Comment:
I've changed it. Also I changed mode from 0640 to 0644 -- it is default mode of each /etc/default* file